modulefiles/gfsutils_gaeac6.intel.lua

```diff
@@ -0,0 +1,32 @@
+help([[
+  This module loads libraries required for building and running GFS UTILS 
+  on the NOAA RDHPC machine Gaea C6 using Intel-2023.1.0.
+]])
+
+whatis([===[Loads libraries needed for building the UFS Weather Model on Gaea C6 ]===])
```

Comment:
```suggestion
whatis([===[Loads libraries needed for building the GFS utilities Gaea C6 ]===])
```

Reply:
@aerorahul I shouldn't miss these. Thank you.